pcap-remote.c

```diff
@@ -2126,7 +2128,7 @@ int rpcap_remoteact_getsock(const char *host, char *errbuf)
 		while (ai_next)
 		{
 			if (sock_cmpaddr(&temp->host, (struct sockaddr_storage *) ai_next->ai_addr) == 0)
-				return (temp->sockctrl);
+				return (int) (temp->sockctrl);
```

Comment:
I cleaned up the code in 8a36b5e79defb6a917d257d81cb986ad350ba881, so this shouldn't be necessary.